serverless/serverless
# Spotinst - Adding documentation for new environment variables feature and update the quick start

## What did you implement:

Adding documentation for environment variables



## How did you implement it:

simple documentation change



## How can we verify it:

check the doc/providers/spotinst/ : 
Readme.md
guide/variables.md
guide/quick-start.md



## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
